src/hooks/eventData.js

```diff
@@ -0,0 +1,39 @@
+import { graphql, useStaticQuery } from "gatsby";
+
+const getEventData = () => {
+  const {
+    allMarkdownRemark: {
+      edges: [
+        {
+          node: {
+              frontmatter: { date, description, photo, title },
+          },
+        },
+      ],
+    },
+  } = useStaticQuery(graphql`
+    query eventDataQuery {
+      allMarkdownRemark(filter: {fileAbsolutePath: {regex: "\\/event/"}}) {
+        edges {
+          node {
+            frontmatter {
+              date
+              description
+              photo
+              title
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  return {
+    date,
+    description,
+    photo,
+    title,
+  };
```

Comment:
This doesn't work for multiple events. If there are two event .md files, how would the user access this information? 

Return an array of objects that return these fields.